### PR TITLE
App manager v2: Fix v1/v2 include mixup

### PR DESCRIPTION
**Pull request by orangejenny**
@dannyroberts / @proteusvacuum 

Used "question" in the text because the column is titled "question". Open to further discussion on what the best copy is.

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_ko_templates.html [hunk 1]
```diff
@@ -130,7 +130,7 @@ <h4 class="panel-title panel-title-nolink">
 <script type="text/html" id="case-config:case-transaction:case-properties">
     <div class="panel-heading">
       <h4 class="panel-title panel-title-nolink">
-        <i class="fa fa-save"></i> {% trans "Save Question Answers to Case Properties" %}
```
Comment (proteusvacuum) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_ko_templates.html]: Can you also change the column in the table below?

Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_ko_templates.html]: Oh, nevermind, that's not what you're saying at all.

Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_ko_templates.html]: Yeah, I'll update the column name.

Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_ko_templates.html]: https://github.com/dimagi/commcare-hq/pull/15969/commits/1e510e57811c0bb34ed78506560a792bf9ab886a

**Review by proteusvacuum — APPROVED** [review]
Thanks!

**Pull request merged**